### PR TITLE
Adding events

### PR DESCRIPTION
**Pull request by WalAmeni**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Adding business logger in swe process

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Review by schwitzguebel — APPROVED** [review]
Tres longue PR, aurait peut etre du etre decoupé ?

File: gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyParallelization.java [hunk 1]
```diff
@@ -19,25 +21,40 @@
...
 @Service
 public class DichotomyParallelization {
-
+    private final Logger businessLogger;
     private final DichotomyLogging dichotomyLogging;
     private final DichotomyRunner dichotomyRunner;
     private final OutputService outputService;
     private final VoltageCheckService voltageCheckService;
     private final CneFileExportService cneFileExportService;
+    public static final String SUMMARY = "Summary [{}] :  " +
```
Comment (TheoPascoli) [inline gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyParallelization.java:43]: Could this be relocated to the dichotomyLogging class ? So this class does not perform anything related to building logs

File: gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyParallelization.java [hunk 2]
```diff
@@ -78,4 +98,29 @@ private SweDichotomyResult getDichotomyResultByDirection(ExecutionResult<SweDich
         }
         return result.get();
     }
+
+    private void generateSummaryEvents(DichotomyDirection direction, DichotomyResult<RaoResponse> dichotomyResult, SweData sweData, Optional<VoltageMonitoringResult> voltageMonitoringResult) {
```
Comment (TheoPascoli) [inline gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyParallelization.java:102]: same, maybe put this to dichotomy logging class ?

File: gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/domain/SweData.java [hunk 3]
```diff
@@ -74,4 +76,12 @@ public String getJsonCracPathEsPt() {
     public String getJsonCracPathFrEs() {
         return jsonCracPathFrEs;
     }
+
+    public Crac getCrac(DichotomyDirection direction) {
```
Comment (TheoPascoli) [inline gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/domain/SweData.java:80]: I dont like having logic to domain class, they should just be storage class. The logic should be done when used in my opinion. To discuss

File: gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/NetworkService.java [hunk 4]
```diff
@@ -131,16 +137,18 @@ private void deleteFile(File srcFile) {
...
-        LOGGER.info("HVDC added to network");
+        List<String> hvdcIds = params.getHvdcCreationParametersSet().stream().map(HvdcCreationParameters::getId).collect(Collectors.toList());
+        LOGGER.info("HVDC {} added to network", hvdcIds);
+        businessLogger.info("HVDC {} added to network", hvdcIds);
     }
 
     private void addPst(Network network) {
         try {
             network.getTwoWindingsTransformer("_e071a1d4-fef5-1bd9-5278-d195c5597b6e").getPhaseTapChanger().setRegulationMode(PhaseTapChanger.RegulationMode.FIXED_TAP);
             network.getTwoWindingsTransformer("_7824bc48-fc86-51db-8f9c-01b44933839e").getPhaseTapChanger().setRegulationMode(PhaseTapChanger.RegulationMode.FIXED_TAP);
```
Comment (TheoPascoli) [inline gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/NetworkService.java:148]: replace those strings with the constants you defined above

File: gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyParallelization.java (hunk 1 above)
Comment (WalAmeni) [reply gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyParallelization.java:43]: Yes I agree

**Pull request merged**